src/ansys/geometry/core/plotting/trame_gui.py

```diff
@@ -32,36 +42,64 @@
 
 
 class TrameVisualizer:
-    """Defines the trame layout view."""
+    """
+    Defines the trame layout view.
 
-    def __init__(self) -> None:
+    Parameters
+    ----------
+    client_type : str, optional
+        Client type to use for the trame server. Options are 'vue2' and 'vue3'.
+        Default is 'vue3'.
+    """
+
+    def __init__(self, client_type: str = "vue3") -> None:
         """Initialize the trame server and server-related variables."""
         if not _HAS_TRAME:  # pragma: no cover
             raise ModuleNotFoundError(
                 "The package 'pyvista[trame]' is required to use this function."
             )
+        self._client_type = client_type
+        if client_type not in ["vue2", "vue3"]:
+            self._client_type = "vue3"
+            logger.warning("Invalid client type {}. Defaulting to 'vue3'.", client_type)
 
-        self.server = get_server()
+        self.server = get_server(client_type=client_type)
         self.state, self.ctrl = self.server.state, self.server.controller
 
-    def set_scene(self, plotter):
+    def set_scene(self, plotter: Union["pv.Plotter", Plotter, "PlotterHelper"]):
         """
         Set the trame layout view and the mesh to show through the PyVista plotter.
 
         Parameters
         ----------
-        plotter : ~pyvista.Plotter
-            PyVista plotter with the rendered mesh.
+        plotter : Union[~pyvista.Plotter, ~Plotter, ~PlotterHelper]
+            PyVista plotter or PyAnsys plotter to render.
         """
+        if isinstance(plotter, Plotter):
+            plotter = plotter.scene
+
+        from ansys.geometry.core.plotting.plotter_helper import PlotterHelper
+
+        if isinstance(plotter, PlotterHelper):
+            pv_plotter = plotter._pl.scene
+        elif isinstance(plotter, Plotter):
+            pv_plotter = plotter.scene
+        else:
+            pv_plotter = plotter
+
```

Comment:
Something seems off... don't you need to delete the first if? Also, aren't you importing the pyvista plotter to do a type check here?

Reply:
I would have to import PyVista to be able to use `isinstance`, but no big deal. I'll import the PyVista in the module.